arelle/XmlValidateParticles.py

```diff
@@ -11,7 +11,13 @@
 from arelle.ModelObject import ModelObject, ModelAttribute
 from arelle.XmlValidate import validate
 
+excluded_schema = [
+    'XMLSchema.xsd'
+]
+
 def validateElementSequence(modelXbrl, compositor, children, ixFacts, iNextChild=0):
+    if compositor.modelDocument.basename in excluded_schema:
```

Comment:
This `in` feels too lose, and checking the XSD at all seems a bit too loose.  Can we instead check the namespace for equality?  I would probably reframe it as a check against "builtin" rather than "excluded".